### DISCUSSION TITLE
Regular Dropouts

### DISCUSSION THREAD

**Discussion by Johnliquid** [apohl79/audiogridder#267] [Q&A]
I'm using 1.1 (Beta 10). With two PCs networked using VNC. It's the first time I've networked two machines together but I don't believe I've done anything wrong. They're networked using a Gigabit switch.

When I run Audiogridder, the audio very regularly stops and the splash screen that shows in the top right overlaid on Cubase, detailing how many instances of Audiogridder plugin and what plugins are running over the network, disappears for a second (whilst the audio stops) and then quickly reappears (alongside the audio restarting) which suggests to me the connection is being dropped.

It does settle down for a time, but my biggest concern is it dropping out during bouncing so I may only get the effects partially applied when bouncing?

**Comment by CaBuffalo** [comment]
This sounds like a network communication problem between your two PCs. 

Is it possible for you to run AG without VNC? This could isolate the cause of the dropouts.

**Comment by Johnliquid** [comment]
Thanks for the response. This is the first time I've ever networked two PCs. 

How would one go about it not using VNC?

**Comment by CaBuffalo** [comment]
I assume you are using 1 set of keyboard/mouse and 1 monitor to control two
PCs.

If so, can you run two PCs separately by plug/unplug your
keyboard/mouse/monitor? After you have the server set up, just leave it on.
You operate on your DAW PC and see if the dropouts happen.

On Wed, Dec 16, 2020 at 8:34 AM Johnliquid <notifications@github.com> wrote:

> Thanks for the response. This is the first time I've ever networked two
> PCs.
>
> How would one go about it not using VNC?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/267#issuecomment-746593696>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AIDE4FXLC5IJYTMZDSVDOSHANCNFSM4U5SP5NQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by Johnliquid** [comment]
Ok cool, and just run a single LAN line between the two ethernet connections?

**Comment by CaBuffalo** [comment]
Yes, you can do it if you have a crossover cable. Otherwise, use the
switch. Please make sure your PCs can see each other in your LAN.

Do you have a router BTW? Connecting to the router makes things a lot
easier.

On Wed, Dec 16, 2020 at 8:41 AM Johnliquid <notifications@github.com> wrote:

> Ok cool, and just run a single LAN line between the two ethernet
> connections?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/267#issuecomment-746605203>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BIHCILD3GCSQYPMZTSVDPNPANCNFSM4U5SP5NQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
